### PR TITLE
Project Chatbot Bully Bot!

### PR DESCRIPTION
**Pull request by OscarSindemark**
This project was fun!

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,28 +1,43 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const form = document.getElementById('form')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
```
Comment (PerErikAndreas) [inline code/script.js:5]: `const input = document.getElementById('input')` Hello Oscar! I know you said you hade some problems with these variables and that the inputWrapper solved it for you so I think you could delete this code when its not used (from what I can see).

File: code/script.js [hunk 2]
```diff
@@ -33,19 +48,135 @@ const showMessage = (message, sender) => {
...
+
+
 // Set up your eventlisteners here
 
+
+inputWrapper.addEventListener('submit', handleNameInput);
+inputWrapper.addEventListener('button', showYesNo)
+inputWrapper.addEventListener('button', lunchMoney)
+
 // When website loaded, chatbot asks first question.
```
Comment (PerErikAndreas) [inline code/script.js:175]: For even more clarity I think you could maybe place these comments in the place where it happens in the code.

File: code/script.js [hunk 3]
```diff
@@ -33,19 +48,135 @@ const showMessage = (message, sender) => {
...
+
+  const No = () => {
+    showMessage("Nope! I chose it for myself.", 'user')
+    showMessage("What a weirdo!", 'bot')
+    setTimeout(lunchMoney, 500)
+}
+
+
+
+// Third interaction between bot and user
```
Comment (PerErikAndreas) [inline code/script.js:102]: These comments makes it clear as you are scrolling down!

**Review by PerErikAndreas — APPROVED** [review]
Hello Oscar!
First of all I must say this bully-theme is hilarious. Great work on that!
Maybe write something in your Info-file and tell us more about how you worked on this project :).

Review:
The page showed up properly and did not crash or act strange in any situation that I could see. The requirements was to make a functioning chatbot with a theme of choice
and with forms of choice, and IF&IF ELSE-statements, and you made that.

The code is well-structured and starts with declaring the variables and ends with the listing the Eventlistener. Nice and clear! You make descriptive notes on places in the code were there’s new interactions between the bot and user .

I know from the demo that you said you had some problems with the variables. I tried to look into that and I think I found a piece of code that we could try to delete. (See the comments).
Nice work!

**Review by RegSod — COMMENTED** [review]
Nice and neat code. Easy to follow.

**Pull request closed without merging**